### PR TITLE
stdin object introduced

### PR DESCRIPTION
**Pull request by Graur**
#343

### CODE REVIEW

**Comment by Graur** [comment]
@yegor256 please, have a look

File: eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * @since 0.23
+ */
+public class EOstdin$EOφ extends PhDefault {
+
+    public EOstdin$EOφ(final Phi parent) {
+        super(parent);
+        this.add("φ", new AtComposite(this, rho -> {
+            try (BufferedInputStream bis = new BufferedInputStream(System.in);
+                 ByteArrayOutputStream buf = new ByteArrayOutputStream()) {
+                int result = bis.read();
```
Comment (yegor256) [inline eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOφ.java:49]: @Graur here is code duplication, you do `bis.read()` twice. A better design:

```
while (true) {
  x = read;
  if (x == -1) {
    break
  }
} 
```

File: eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    @AfterAll
+    public static void restoreSystemInput() {
+        System.setIn(DEFAULT_STDIN);
+    }
+
+    @Test
+    public void nextLineOneLineTest() {
+        String expected = "this is a test input!";
+        mockSystemIn(expected + "\n");
+
```
Comment (yegor256) [inline eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java:56]: @Graur read this: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

**Comment by yegor256** [comment]
@Graur see above

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Graur thanks!

**Pull request merged**